onnxruntime/core/providers/cpu/math/element_wise_ops.cc

```diff
@@ -517,6 +519,7 @@ struct Max_8::ComputeImpl {
 };
 
 Status Max_8::Compute(OpKernelContext* context) const {
+  ORT_ENFORCE(nullptr != context, "Context must not be NULL");
```

Comment:
Are these the only Compute functions that showed up in the warnings? There are many that don't check context against null.

Reply:
Yes - only those showed up here have Xcode warnings.

Reply:
Removed.